lib/api/initiateMultipartUpload.js

```diff
@@ -134,7 +134,16 @@ function initiateMultipartUpload(authInfo, request, log, callback) {
                 }
                 services.metadataStoreMPObject(MPUbucket.getName(),
                     cipherBundle, metadataStoreParams,
-                    log, err => callback(err, xml));
+                    log, err => {
+                        if (err) {
+                            log.trace('error storing multipart object', {
+                                error: err,
+                            });
+                            return callback(err);
+                        }
+                        log.trace('successfully initiated mpu', { uploadId });
```

Comment:
Yes, we only want to log on success but we need to add this as a default field so that it shows up in the logs after this step. Something like
```js
log.addDefaultFields({ uploadId });
```
This call adds it as a field for all the log entries which are going to be printed after this step.
Sorry, I should have mentioned this in the issue.